cmd/cmd.go

```diff
@@ -16,26 +16,36 @@ import (
 )
 
 const (
+	// Use is the name of the CLI application.
 	Use = "dynatrace-bootstrapper"
 
-	SourceFolderFlag   = "source"
-	TargetFolderFlag   = "target"
-	DebugFlag          = "debug"
+	// SourceFolderFlag is the flag for specifying the source folder.
+	SourceFolderFlag = "source"
+	// TargetFolderFlag is the flag for specifying the target folder.
+	TargetFolderFlag = "target"
+	// DebugFlag is the flag for enabling debug mode.
+	DebugFlag = "debug"
+	// SuppressErrorsFlag is the flag for suppressing errors.
 	SuppressErrorsFlag = "suppress-error"
 )
 
+// New creates a new cobra command for the Dynatrace bootstrapper.
 func New(fs afero.Fs) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:                Use,
-		RunE:               run(fs),
-		FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
```

Comment:
do NOT remove this, why did you remove it?

(if you remove this, if the Operator gives an unknown value, it will break, which breaks forward compatibilty )